widgets/snapshots/src/SnapshotsTable.jsx

```diff
@@ -140,6 +140,7 @@ export default class SnapshotsTable extends React.Component {
 
                     {data.items.map(item => {
                         const isUseful = isSnapshotUseful(item);
```

Comment:
What do you think about changing:
```suggestion
                        const isUsable = isSnapshotUsable(item);
```
?

I know it was not introduced in this PR, but when I look at it now it is misleading.